### DISCUSSION TITLE
Contributors, how do you set up your config for the development build of neovim?

### DISCUSSION THREAD

**Discussion by marcg03** [neovim/neovim#26854] [Q&A]
I was thinking of making a script that starts `nvim` with custom values for `XDG_CONFIG_HOME` and `XDG_DATA_HOME`.  
  
Do you recommend something simpler?  
Do I have to set variables other than `XDG_CONFIG_HOME` and `XDG_DATA_HOME` to have an isolated `nvim`?

**Comment by clason** [comment]
`:h $NVIM_APPNAME`
